lib/smart_answer/calculators/marriage_abroad_data_query.rb

```diff
@@ -8,6 +8,10 @@ class MarriageAbroadDataQuery
 
     FRENCH_OVERSEAS_TERRITORIES = %w(french-guiana french-polynesia guadeloupe martinique mayotte new-caledonia reunion st-pierre-and-miquelon wallis-and-futuna)
 
+    CEREMONY_COUNTRIES_OFFERING_PACS = %w(
+      france monaco new-caledonia wallis-and-futuna
+    )
+
```

Comment:
Minor: The formatting of this array is inconsistent compared to the ones immediately above and below.